server/src/main/scala/org/http4s/server/websocket/package.scala

```diff
@@ -38,17 +38,19 @@ package object websocket {
     *                 are plans to address this limitation in the future.
     * @param status The status code to return to a client making a non-websocket HTTP request to this route
     */
+  @deprecated("Use WebSocketBuilder", "0.18")
   def WS[F[_]](
       send: Stream[F, WebSocketFrame],
       receive: Sink[F, WebSocketFrame],
       status: F[Response[F]])(implicit F: Monad[F]): F[Response[F]] =
-    WebSocketBuilder(
+    WebSocketBuilder[F].build(
       send,
       receive,
       Headers.empty,
       status,
-      Response[F](Status.BadRequest).withBody("WebSocket handshake failed.")).toResponse
+      Response[F](Status.BadRequest).withBody("WebSocket handshake failed."))
 
+  @deprecated("Use WebSocketBuilder", "0.18")
```

Comment:
Shouldn't the version be "0.18.0-M7" ?

Reply:
fixing

Reply:
is it failing because BlazeWebSocketExampleApp uses deprecated WS?